easybuild/easyconfigs/o/orjson/orjson-3.9.15-GCCcore-12.3.0.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'orjson'
+version = '3.9.15'
+
+homepage = 'https://github.com/ijl/orjson'
+description = """Fast, correct Python JSON library supporting dataclasses, datetimes, and numpy."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('maturin', '1.4.0', '-Rust-1.75.0'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('ruff', '0.4.8', {
+        'checksums': ['16d717b1d57b2e2fd68bd0bf80fb43931b79d05a7131aa477d66fc40fbd86268'],
+    }),
+    ('mypy_extensions', '1.0.0', {
+        'checksums': ['75dbf8955dc00442a438fc4d0666508a9a97b6bd41aa2f0ffe9d2f2725af0782'],
+    }),
+    ('mypy', '1.10.0', {
+        'checksums': ['3d087fcbec056c4ee34974da493a826ce316947485cef3901f511848e687c131'],
+    }),
+    (name, version, {
+        'checksums': ['95cae920959d772f30ab36d3b25f83bb0f3be671e986c72ce22f8fa700dae061'],
+    }),
+]
+
+sanity_pip_check = True
+
+moduleclass = 'math'
```

Sorry, one more thing: I don't think this really fits in `math`, since it's not really doing any mathematical stuff. Other JSON libraries/tools seem to be in `lib`, so I guess that's a better fit?
```suggestion
moduleclass = 'lib'
```